ingestion-beam/src/main/resources/account-ecosystem/telemetry.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Format of Account Ecosystem pings from desktop telemetry before decryption",
+  "type": "object",
+  "properties": {
+    "payload": {
+      "type": "object",
+      "properties": {
+        "ecosystemClientId": {
+          "type": "string"
+        },
+        "ecosystemDeviceId": {
+          "type": "string"
+        },
+        "ecosystemAnonId": {
+          "description": "The current Account Ecosystem Telemetry anon_id value for this user; it must be a JOSE JWE objects in Compact Serialization form",
+          "type": "string"
+        },
+        "previousEcosystemAnonIds": {
+          "description": "Previous Account Ecosystem Telemetry anon_id values associated with this user",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        }
+      },
+      "required": [
+        "ecosystemClientId",
+        "ecosystemDeviceId"
```

Comment:
@jklukas I'm lightly embarrassed to be asking this question, but...what's the difference between "client id" and "device id" here?

I couldn't find a definition of "device id" in [the "identifiers" spec doc](https://docs.google.com/document/d/1rR3uJG8sVtow4plYu6M5Jp5e5zOSrLzdkUx8BdO8kOM), but it does confusing talk about the device being identified by a "client id".

Reply:
It's not obvious to me what the difference is here. I'll try to track down the history of docs to see where these two terms were introduced. I'm definitely open to removing ecosystemDeviceId if it's not meaningfully different from client id.

Reply:
I think we should have just `client_id` and remove `device_id`, if that makes sense to you. That's certainly the only such value I had intended on submitting with the first pings from Firefox desktop :-)

Reply:
I have not been able to track down where ecosystem_device_id came from, so I'm inclined to go ahead and remove it. We can easily mark it as non-required immediately, and then I can follow up about recreating the tables to remove it from the schema completely.

Reply:
Thanks, SGTM 👍